config/initializers/0_clowder-config.rb

```diff
@@ -1,19 +1,28 @@
+# frozen_string_literal: true
+
+def build_endpoint_url(endpoint, config)
+  return URI::HTTPS.build(host: endpoint&.hostname, port: endpoint&.tlsPort).to_s if config.tlsCAPath
+
+  URI::HTTP.build(host: endpoint&.hostname, port: endpoint&.port).to_s
+end
 
 if ClowderCommonRuby::Config.clowder_enabled?
 
   config = ClowderCommonRuby::Config.load
 
+  ENV['SSL_CERT_FILE'] = config.tlsCAPath if config.tlsCAPath
```

Comment:
Possibly causes issues with cloudwatch logging but might not be a problem for us because we don't really have cloudwatch fully implemented anyway.